Content.Server/Containers/ThrowInsertContainerComponent.cs

```diff
@@ -0,0 +1,35 @@
+using Robust.Shared.Audio;
+
+namespace Content.Server.Containers;
+
+/// <summary>
+/// Allows objects to fall inside the Container when thrown
+/// </summary>
+[RegisterComponent]
+[Access(typeof(ThrowInsertContainerSystem))]
+public sealed partial class ThrowInsertContainerComponent : Component
+{
+    [DataField(required: true)]
+    public string? ContainerId;
```

Comment:
no reason for this to be null, `string ContainerId = string.Empty` and remove null checks